File: Dropspot/app/src/main/java/com/example/dropspot/data/model/dto/requests/ParkSpotRequest.java
package com.example.dropspot.data.model.dto.requests;

import lombok.Getter;
import lombok.Setter;


@Getter
@Setter
public class ParkSpotRequest {
    private String name;

    private double latitude;

    private double longitude;

    private double entranceFee;

    private boolean isIndoor;

    private String street;

    private String houseNumber;

    private String postalCode;

    private String city;

    private String state;

    private String country;
}

File: Dropspot/app/src/main/java/com/example/dropspot/data/model/dto/requests/ParkSpotRequest.kt
package com.example.dropspot.data.model.dto.requests

import com.example.dropspot.data.model.ParkCategory

data class ParkSpotRequest(
    val name: String,
    val latitude: Double,
    val longitude: Double,
    val entranceFee: Double = 0.0,
    val parkCategory: ParkCategory,
    private val street: String,
    private val houseNumber: String,
    private val postalCode: String,
    private val city: String,
    private val state: String,
    private val country: String
)